src/test/ui/impl-trait/issue-53598.rs

```diff
@@ -0,0 +1,28 @@
+// ignore-tidy-linelength
+#![feature(type_alias_impl_trait)]
```

Comment:
Please move the tests into https://github.com/rust-lang/rust/tree/master/src/test/ui/type-alias-impl-trait.

Reply:
Done